modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/source-builder/serializeUtil.js

```diff
@@ -437,6 +437,9 @@ function appendXMLNotifications(buffer, notifications, nodeName, exporting) {
 				Array.isArray(currentRecipients[index])
 			) {
 				for (const recipientsIndex in currentRecipients[index]) {
+					if (recipientsIndex === 'sectionsData') {
```

Comment:
Hey @cgmonte,
I realized that this is a bug specific to notifications (it doesn't happen with the notification timer). It turns out that we are using the 'sectionsData' position to save the data of the different recipients types, in the cases of Role Type and User we save an array because we can have several sections, in the other cases we save an object. But this data is not saved in the XML because it is not necessary. In the case of the Notification timer the bug doesn't occur because we have an 'auxiliary state' (actionSections) in which we save the 'sectionsData' but when it comes to passing the values to the selectedItem state we only pass what is necessary to save in the XML. In the case of notifications, the values are saved directly in the selectedItem state so it is sent to the serialize function. To solve this, we could change the approach a little and save this data in another state, for example

Reply:
Done!